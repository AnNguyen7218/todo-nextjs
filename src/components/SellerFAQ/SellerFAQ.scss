@import 'config.gutters';
@import 'config.fonts';
@import 'config.media';

.container {
  margin-top: gutter(48);
  margin-bottom: gutter(48);

  @include break(md) {
    display: flex;
    justify-content: space-between;
  }

  @include break(xl) {
    margin-top: gutter(96);
    margin-bottom: gutter(96);
  }
}

.header {
  margin-bottom: gutter(32);

  @include break(md) {
    width: 10%;
  }
}

.help-question {
  margin-top: gutter(40);
  margin-bottom: gutter(8);
  @include small;
}

.FAQ-section {
  @include break(md) {
    width: 70%;
  }
}
