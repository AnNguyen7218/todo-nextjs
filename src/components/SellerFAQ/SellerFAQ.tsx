import React from 'react';
import { FAQ } from 'constants/sellerLandingPage';
import { Container, CollapseGroup, Button } from 'components';
import css from './SellerFAQ.scss';

const SellerFAQ = () => {
  return (
    <Container className={css['container']}>
      <h2 className={css['header']}>FAQ</h2>
      <div className={css['FAQ-section']}>
        <CollapseGroup collapseItems={FAQ} />
        <div>
          <p className={css['help-question']}>
            Can&apos;t find what you&apos;re looking for?
          </p>
          <Button buttonType="fixed" theme="border">
            Help center
          </Button>
        </div>
      </div>
    </Container>
  );
};

export default SellerFAQ;
