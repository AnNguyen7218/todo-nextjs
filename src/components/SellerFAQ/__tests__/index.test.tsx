import React from 'react';
import { render, cleanup } from '@testing-library/react';
import SellerFAQ from '../SellerFAQ';

afterEach(cleanup);

describe('SellerFAQ Component', () => {
  test('SellerFAQ should match snapshot', () => {
    const { container } = render(<SellerFAQ />);
    expect(container).toMatchSnapshot();
  });
});
