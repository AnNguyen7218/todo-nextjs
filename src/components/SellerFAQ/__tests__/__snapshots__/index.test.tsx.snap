// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SellerFAQ Component SellerFAQ should match snapshot 1`] = `
<div>
  <div
    class="container container"
  >
    <h2
      class="header"
    >
      FAQ
    </h2>
    <div
      class="FAQ-section"
    >
      <div
        class="container"
      >
        <div
          class="header-container"
        >
          <h3
            class="title"
          >
            What is Intrepid?
          </h3>
          <ic-add.svg
            classname="icon"
          />
        </div>
        <div
          class="content"
          style="height: 0px; margin-top: 0px;"
        >
          <p>
            Intrepid is a global business-to-business e-commerce marketplace to buy and sell professional technology and creative services.
          </p>
        </div>
      </div>
      <div
        class="container"
      >
        <div
          class="header-container"
        >
          <h3
            class="title"
          >
            Who is Intrepid for?
          </h3>
          <ic-add.svg
            classname="icon"
          />
        </div>
        <div
          class="content"
          style="height: 0px; margin-top: 0px;"
        >
          <p>
            If youâ€™re a top design, technology, data science or digital marketing studio, we welcome you to start selling your services on the Intrepid marketplace.
          </p>
        </div>
      </div>
      <div>
        <p
          class="help-question"
        >
          Can't find what you're looking for?
        </p>
        <button
          class="button button--fixed button--border"
          type="button"
        >
          <div
            class="button-children"
          >
            Help center
          </div>
        </button>
      </div>
    </div>
  </div>
</div>
`;
